tests/unit/s2n_record_size_test.c

```diff
@@ -227,6 +227,31 @@ int main(int argc, char **argv)
             EXPECT_LESS_THAN_EQUAL(bytes_written, RECORD_SIZE_LESS_OVERHEADS);
         }
 
+        /* TLS1.3 AEAD */
```

Comment:
Why does the diff show it as you are adding a new test here? I dont see the removal/modification to the older test; s2n_record_size_test.c has a version of this test here: https://github.com/aws/s2n-tls/blob/395da6528e8758ca7e68f6647d8bafc93afb21d5/tests/unit/s2n_record_size_test.c#L208-L228

Reply:
https://github.com/aws/s2n-tls/pull/2900#discussion_r652048306